scripts/pubtator_to_xml.py

```diff
@@ -126,17 +127,15 @@ def read_bioconcepts2pubtator_offsets(path):
         # Convert "illegal chracters" (i.e. < > &) in the main text
```

Comment:
These comments look leftover from past code? I think we should remove them in this PR, since we're working on this section.